color/src/x11_colors.rs

```diff
@@ -312,15 +312,15 @@ fn weak_hash(key: u32, salt: u32, n: usize) -> usize {
     (((y as u64) * (n as u64)) >> 32) as usize
 }
 
-pub(crate) fn lookup_palette(s: &str) -> Option<[u8; 4]> {
+pub(crate) fn lookup_palette(s: &str) -> Option<usize> {
```

Comment:
Could you rename this to `lookup_palette_index` or similar? Maybe even with css_palette? Or should this not be duplicated and moved to the palette code with a single function definition?

Reply:
The rename: e227926c3ee048ea8afd8698b34ece04bd3a146e, 013c90b7c2b92d41c8b673d81bd2321da803c9fe.

Moving it around is a bit more difficult as the file is auto-generated, but I'm unsure what you mean by duplication of the function.